src/ElectricPokemon.java

```diff
@@ -0,0 +1,52 @@
+public class ElectricPokemon extends Pokemon {
+    int volt;
+    int accurancy;
+
+    public ElectricPokemon(String name, int volt, int accurancy) {
+        super(name);
+        this.volt = volt;
+        this.accurancy = accurancy;
+        this.setType("Electric");
```

Comment:
Je roept in elke subklasse de setType() methode aan, waarom zet je dat niet in de constructor van Pokemon?